data/sdf/test/test_skeleton_joint.world

```diff
@@ -0,0 +1,278 @@
+<?xml version="1.0" ?>
+<sdf version="1.4">
```

Comment:
```suggestion
<sdf version="1.5">
```

Looking at the [specification](http://sdformat.org/spec?elem=joint&ver=1.5), the new joint dynamics properties were introduced in version 1.5.

Reply:
@jslee02 Resolved the conflicts and modified the sdf file version to 1.5